<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Sandeeps-MacBook-Air.local" ignored="0" name="OpenAccountTest" tests="3" failures="3" timestamp="15 Dec 2018 21:54:14 GMT" time="44.843" errors="0">
  <testcase name="openAccountTest" time="42.41" classname="com.w2a.suite.bankmanager.testcases.OpenAccountTest">
    <failure type="java.lang.AssertionError" message="Failing while clicking on the element process_CSS">
      <![CDATA[java.lang.AssertionError: Failing while clicking on the element process_CSS
	at com.w2a.datadriven.base.TestBase.reportFailure(TestBase.java:192)
	at com.w2a.datadriven.base.TestBase.click(TestBase.java:214)
	at com.w2a.suite.bankmanager.testcases.OpenAccountTest.openAccountTest(OpenAccountTest.java:32)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- openAccountTest -->
  <testcase name="openAccountTest" time="42.865" classname="com.w2a.suite.bankmanager.testcases.OpenAccountTest">
    <failure type="java.lang.AssertionError" message="Failing while clicking on the element process_CSS">
      <![CDATA[java.lang.AssertionError: Failing while clicking on the element process_CSS
	at com.w2a.datadriven.base.TestBase.reportFailure(TestBase.java:192)
	at com.w2a.datadriven.base.TestBase.click(TestBase.java:214)
	at com.w2a.suite.bankmanager.testcases.OpenAccountTest.openAccountTest(OpenAccountTest.java:32)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- openAccountTest -->
  <testcase name="openAccountTest" time="43.84" classname="com.w2a.suite.bankmanager.testcases.OpenAccountTest">
    <failure type="java.lang.AssertionError" message="Failing while clicking on the element process_CSS">
      <![CDATA[java.lang.AssertionError: Failing while clicking on the element process_CSS
	at com.w2a.datadriven.base.TestBase.reportFailure(TestBase.java:192)
	at com.w2a.datadriven.base.TestBase.click(TestBase.java:214)
	at com.w2a.suite.bankmanager.testcases.OpenAccountTest.openAccountTest(OpenAccountTest.java:32)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- openAccountTest -->
</testsuite> <!-- OpenAccountTest -->
